### DISCUSSION TITLE
Can I run gotenberg on i386 machine running Debian Linux?

### DISCUSSION THREAD

**Discussion by kmandalas** [gotenberg/gotenberg#353] [Q&A] [answered]
Hello and kudos for this excellent multi-purpose tool you have provided to the community.

My question is: can I run gotenberg on i386 machine running Debian Linux?
I have installed Docker but the gotenberg  container does not seem to be able to start.

Thank you.

**Comment by gulien** [comment, marked as answer]
Hello @kmandalas 👋 

As explained in #338, I don't necessarily want to maintain too many architectures, as it's hard to test them. For instance, GitHub Actions does not provide an `arm64` runtime.

Currently, only `amd64` and `arm64` architectures are available, but do not hesitate to add your request to the previous issue 😄

**Reply by kmandalas** [reply]
Hi @gulien - thanks so much for your prompt response. One more thing: is it possible for me to run it, not dockerized but as a Go executable?

**Reply by gulien** [reply]
I guess it should be possible 🤔 but there might be a lot of unknown caveats to be honest. IMO the best option is to stick to the Docker image, as it bundles everything required for running Gotenberg 😄

**Reply by kmandalas** [reply]
If I am interested only in docx to PDF conversion what else do I need besides to: clone, build and run gotenberg? I guess what is in https://github.com/gotenberg/gotenberg/blob/main/build/Dockerfile ?

**Reply by gulien** [reply]
Indeed, but there might be some issue related to the package https://github.com/gotenberg/gotenberg/blob/v7.0.6/pkg/gotenberg/cmd.go (see also  https://medium.com/@felixge/killing-a-child-process-and-all-of-its-children-in-go-54079af94773), which only works on Linux AFAIK.
